ipd/0024/README.md

```diff
@@ -50,6 +245,13 @@ At present the port boots via bespoke booter derived from `inetboot` by
 [Hayashi Naoyuki](https://github.com/n-hys), and delivered per target
 platform.
 
+As part of UEFI and ACPI exploration, the illumos UEFI loader has been [ported
+to aarch64](
+https://github.com/r1mikey/illumos-gate/commit/be9a9ee13d660819202677b912dee60cb3b38613)
+(tested on the Qemu sbsa-ref board) and works with minimal architecture changes.
+This work is minimally usable but makes expedient decisions around how to boot
+that have not been widely discussed.
```

Comment:
The discussion in general here has been me and @tsoome wondering exactly what we should do, agreeing probably loader should read and interpret the kernel ELF rather than needing something like dboot if we can possibly avoid it, and then not having time to go further

Reply:
I ended up going with the FreeBSD structures and a PIE unix.shim - loader loads unix.shim (built from a lot of the same code as inetboot), unix and the boot archive, then jumps to the shim, which sets up the kernel at the desired virtual address and jumps to it.

When boot memory is cleaned up the shim disappears.

It's not particularly elegant, but the end result works well. Still lots of rough edges though.